packages/core/admin/admin/src/pages/Settings/pages/Webhooks/ListPage.tsx

```diff
@@ -383,7 +384,8 @@ const ListPage = () => {
                 <Button
                   variant="secondary"
                   startIcon={<Plus />}
-                  onClick={() => (canCreate ? goTo('create') : {})}
+                  disabled={!canCreate}
+                  onClick={canCreate ? goTo('create') : undefined}
```

Comment:
is this check necessary? the disabled flag should stop the callback right?

Reply:
Yes, but Typescript fights back

Reply:
Does it? I've just tried locally and can't see an issue at the moment 🤔  can you share what you see?

Reply:
Nevermind, again